packages/core/src/App/Containers/VerificationModal/verification-modal-content.tsx

```diff
@@ -1,4 +1,243 @@
 import React from 'react';
+import { Div100vhContainer } from '@deriv/components';
+import { isDesktop } from '@deriv/shared';
+import { LandingCompany, ResidenceList, GetSettings, StatesList, GetAccountStatus } from '@deriv/api-types';
+import { ProofOfIdentityContainer, ProofOfAddressContainer } from '@deriv/account';
+import { connect } from 'Stores/connect';
+import RootStore from 'Stores/index';
+
+type TAuthenticationStatus = { document_status: string; identity_status: string };
+
+type TStoreProofOfAddressArgs = {
+    file_uploader_ref: HTMLDivElement | null;
+    values: { [key: string]: string };
+};
+
+type TRemoveNotificationMessage = {
+    key: string;
+    should_show_again: boolean;
+};
+
+type TGetSettings = GetSettings & {
+    upload_file?: string;
+    poi_state?: string;
+};
+
+type TVerificationModalContent = {
+    fetchResidenceList: () => void;
+    getChangeableFields: () => void;
+    is_mx_mlt: boolean;
+    is_switching: boolean;
+    is_high_risk: boolean;
+    is_withdrawal_lock: boolean;
+    should_allow_authentication: boolean;
+    is_virtual: boolean;
+    updateAccountStatus: () => void;
+    is_eu: boolean;
+    is_verification_modal_visible: boolean;
+    addNotificationByKey: (key: string) => void;
+    authentication_status: TAuthenticationStatus;
+    account_settings: TGetSettings;
+    client_email: string;
+    context: RootStore;
+    is_fully_authenticated: boolean;
+    landing_company: LandingCompany;
+    has_restricted_mt5_account: () => void;
+    refreshNotifications: () => void;
+    removeNotificationMessage: () => void;
+    removeNotificationByKey: (args: TRemoveNotificationMessage) => void;
+    residence_list: ResidenceList;
+    states_list: StatesList;
+    storeProofOfAddress: TStoreProofOfAddressArgs;
+    fetchStatesList: () => void;
+    account_status: GetAccountStatus;
+    app_routing_history: any;
```

Comment:
can you check the type declared here?
packages/shared/src/utils/platform/platform.ts